code/modules/mob/living/carbon/human/life.dm

```diff
@@ -794,6 +794,21 @@
 
 	var/body_temperature_difference = species.body_temperature - bodytemperature
 
+	var/hypotemp = min(species.cold_level_1+17.5,T0C+10) //People going into hypothermia in 20C might be a bit overboard.

```

Comment:
why do you arbitrarily make hypothermia trigger *above* cold_level_1? x3

Reply:
Because cold levels are absolutely borked at the moment. Humans can stand outside in -30C weather in a blizzard, naked, and chill.

Reply:
Thats issue with the comfort levels and its functionality. Starting hypothermia before the damage/warnings are supposed to start coming in sounds like just bad design.

Reply:
> Thats issue with the comfort levels and its functionality. Starting hypothermia before the damage/warnings are supposed to start coming in sounds like just bad design.

That's why I also added an alert for it. Also nobody's going to want to redo the temperature system so in my mind this is the real only way of making things sane without bursting someone's bubble. Plus it makes the whole thing a lot more interesting in general